docs/architecture/raft.rst

```diff
@@ -309,42 +306,42 @@ Examples
      - Try restarting the nodes. If the nodes are dead, :doc:`replace them with new nodes </operating-scylla/procedures/cluster-management/replace-dead-node-or-more/>`.
    * - 1 DC
-     - Schema updates are possible and safe.
+     - Schema and toplogy updates are possible and safe.
      - When the DC comes back online, try restarting the nodes in the cluster. If the nodes are dead, :doc:`add 3 new nodes in a new region </operating-scylla/procedures/cluster-management/add-dc-to-existing-dc/>`.
    * - 2 DCs
-     - Cluster is not fully operational. The data is available for reads and writes, but schema changes are impossible.
-     - When the DCs come back online, restart the nodes. If at least one DC fails to come back online and the nodes are lost, consult the :ref:`manual Raft recovery section <recover-raft-procedure>`.
+     - Data is available for reads and writes, but schema and toplogy changes are only possible once the quorum is restored.
+     - When the DCs come back online, restart the nodes. If at least one DC fails to come back online and the nodes are lost, consult the :ref:`Raft recovery section <recover-raft-procedure>`.
 
 .. _recover-raft-procedure:
 
-Raft manual recovery procedure
-==============================
+Raft Recovery Procedure
+=======================
 
 .. versionadded:: 5.2
 
-The manual Raft recovery procedure applies to the following situations:
+Prerequisite
+~~~~~~~~~~~~
+                  
+The Raft recovery procedure applies to the following situations:
 
+* A majority of nodes failed and are **irrecoverable**. Raft can only progress if a majority (quorum) of nodes are available. If it is possible to restart your nodes, do that instead of the recovery procedure.
 * :ref:`The internal Raft upgrade procedure <verify-raft-procedure>` got stuck because one of your nodes failed in the middle of the procedure and is irrecoverable,
-* or the cluster was running Raft but a majority of nodes (e.g. 2 our of 3) failed and are irrecoverable. Raft cannot progress unless a majority of nodes is available.
-
-.. warning::
 
-   Perform the manual recovery procedure **only** if you're dealing with **irrecoverable** nodes. If it is possible to restart your nodes, do that instead of manual recovery.
 
-.. warning::
-
-   Before proceeding, make sure that the irrecoverable nodes are truly dead, and not, for example, temporarily partitioned away due to a network failure. If it is possible for the 'dead' nodes to come back to life, they might communicate and interfere with the recovery procedure and cause unpredictable problems.
+Make sure that the dead nodes are irrecoverable, and are not, for example, temporarily partitioned away due to a network failure. If "dead" nodes return to life, they might communicate and interfere with the recovery procedure.
+It's good practice to ensure that using firewall rules or otherwise isolates your live nodes and blocks communication from these "dead" nodes.
```

Comment:
@kostja @asias  is this still relevant?
https://github.com/scylladb/scylladb/commit/12ab2c3d8d6925a14749be61b90a3032c994f1f8 Prevent removed node to restart and join the cluster